Library/Homebrew/cmd/bottle.rb

```diff
@@ -95,10 +95,10 @@ def print_filename(string, filename)
       end
     end
 
-    keg_contains_absolute_symlink_starting_with?(string, keg) || result
+    keg_contains_absolute_symlink_starting_with(string, keg) || result
```

Comment:
`result || keg...` feels slightly more idiomatic and avoids calling the function if unnecessary.